### PR TITLE
Newsletter-153:Translate into Japanese

### PR DESCRIPTION
**Pull request by azuchi**
To translate the field report, I created a translate file under the `_includes/articles/ja` directory. Please point out if there is another preferred place.

### CODE REVIEW

File: _posts/ja/2021-06-16-cardcoins-rbf-batching-ja.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+title: 'Field Report: Using RBF and Additive Batching'
+permalink: /ja/cardcoins-rbf-batching/
+name: 2021-06-16-cardcoins-rbf-batching-ja
+slug: 2021-06-16-cardcoins-rbf-batching-ja
+type: posts
+layout: post
+lang: ja
+version: 1
+excerpt: >
+   "Additive batching" is a scheme whereby additional outputs are
```
Comment (bitschmidty) [inline _posts/ja/2021-06-16-cardcoins-rbf-batching-ja.md:11]: @azuchi can you translate this as appropriate?

**Review by bitschmidty — COMMENTED** [review]
@azuchi I added a commit which adds the article to the Japanese blog list as well (https://bitcoinops.org/ja/blog/)

**Comment by bitschmidty** [comment]
@harding Do you have any objection to the Japanese translation for the field report article being in a new _includes/articles/ja/ folder?

File: _posts/ja/2021-06-16-cardcoins-rbf-batching-ja.md (hunk 1 above)
Comment (azuchi) [reply _posts/ja/2021-06-16-cardcoins-rbf-batching-ja.md]: @bitschmidty Thanks, I translated (cd85de7).

**Comment by bitschmidty** [comment]
@azuchi  looks good to me, thanks for the translations! squashed the last commits.

**Pull request merged**